.bazelrc

```diff
@@ -137,6 +137,11 @@ build --experimental_inprocess_symlink_creation
 test --test_tag_filters=-docker
 build --build_tag_filters=-docker
 
+# Don't build or run Firecracker tests by default, because they cannot be run on all environments
+# (Can only be run on Linux machines with bare execution)
+test --test_tag_filters=-bare
```

Comment:
do these values accumulate? e.g. does `--test_tag_filters=-bare --test_tag_filters=-docker` mean the same thing as `--test_tag_filters=-bare,-docker` ?

Reply:
Hmm good point. Just combined them for simplicity